src/polar.rs

```diff
@@ -97,7 +97,7 @@ impl<PD: PointData> PolarCoordinates for Point<PD> {
     }
 }
 
-impl<PD: PointData> PolarCoordinates for dyn MFEKPointCommon<PD> {
+impl<PD: PointData> PolarCoordinates for &mut dyn MFEKPointCommon<PD> {
```

Comment:
I'm OK with this for now given we can't think of a better way to do it. If @MatthewBlanchard does, he can PR on top of this.